osgp/protocol-adapter-iec60870/osgp-protocol-adapter-iec60870/src/main/java/org/opensmartgridplatform/adapter/protocol/iec60870/domain/valueobjects/RequestInfo.java

```diff
@@ -5,58 +5,56 @@
  *
  *     http://www.apache.org/licenses/LICENSE-2.0
  */
-package org.opensmartgridplatform.adapter.protocol.iec60870.infra.networking.helper;
+package org.opensmartgridplatform.adapter.protocol.iec60870.domain.valueobjects;
 
 import java.io.Serializable;
 
 import org.opensmartgridplatform.shared.infra.jms.MessageMetadata;
 
 /**
- * A value object, containing all data of an incoming ObjectMessage.
+ * A value object, containing the relevant information of an incoming request:
+ * <ul>
+ * <li>Correlation Uid</li>
+ * <li>Organisation Identification</li>
+ * <li>Device Identification</li>
+ * <li>Message Type</li>
+ * <li>IP Address</li>
+ * <li>Domain</li>
+ * <li>Domain Version</li>
+ * </ul>
+ *
  */
-public class RequestMessageData {
+public class RequestInfo {
 
     final Serializable messageData;
     final String domain;
     final String domainVersion;
     final String messageType;
-    final int retryCount;
```

Comment:
Where are the deleted members stored in the new situation?

Reply:
They are currently not used in this protocol adapter,